spec/requests/api/v1/user_spec.rb

```diff
@@ -33,6 +33,40 @@
       expect(attributes).to have_key(:long)
       expect(attributes).to have_key(:phone)
     end
+
+    xit 'lets you send a text' do #used to test initial sign up text 
```

Comment:
I don't know about how to test the texting functionality. Have google searches brought up any type of results on this?